tests/import_export_tests.py

```diff
@@ -145,8 +145,12 @@ def assert_dash_equals(self, expected_dash, actual_dash, check_position=True):
         self.assertEquals(expected_dash.slug, actual_dash.slug)
         self.assertEquals(expected_dash.dashboard_title, actual_dash.dashboard_title)
         self.assertEquals(len(expected_dash.slices), len(actual_dash.slices))
-        expected_slices = sorted(expected_dash.slices, key=lambda s: s.slice_name or "")
-        actual_slices = sorted(actual_dash.slices, key=lambda s: s.slice_name or "")
+        expected_slices = sorted(
```

Comment:
Why not just use the `slice.id`, i.e., `expected_slices = sorted( expected_dash.slices, key=lambda s: s.id)`.